pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -846,6 +846,31 @@ rec {
     };
   };
 
+  /* INDONESIA */
+
+  id_ID = mkDict rec {
```

Comment:
Have you tried `mkDictFromLibreOffice`? I think that's the recommended approach. See the other dictionaries in the file.

Reply:
Done, thank you for the assist @FliegendeWurst